test/example_test.dart

```diff
@@ -1,28 +1,26 @@
-import 'dart:convert';
```

Comment:
do we still need this file?

Reply:
No, but I think @Showcas had removed it in his PR. So I did not want to create any more conflicts between the files.

Reply:
yes a reviewed that after yours my mistkae